import-export-cli/cmd/removeEnv.go

```diff
@@ -83,7 +83,7 @@ func removeEnv(envName, mainConfigFilePath, envKeysFilePath string) error {
 		if store.Has(envName) {
 			err = runLogout(envName)
 			if err != nil {
-				return err
+				utils.Logln("Log out is unsuccessful. ", err)
```

Comment:
Is this space at the end of the sentence expected?

Reply:
Fixed with c672258dbfed4a533e8a19dc8f1ef5050d4bd02e